pkg/proc/core/core.go

```diff
@@ -160,15 +160,20 @@ type Thread struct {
 	common proc.CommonThread
 }
 
-// ErrWriteCore is returned when attempting to write to the core
-// process memory.
-var ErrWriteCore = errors.New("can not write to core process")
+var (
+	// ErrWriteCore is returned when attempting to write to the core
+	// process memory.
+	ErrWriteCore = errors.New("can not write to core process")
 
-// ErrShortRead is returned on a short read.
-var ErrShortRead = errors.New("short read")
+	// ErrShortRead is returned on a short read.
+	ErrShortRead = errors.New("short read")
 
-// ErrContinueCore is returned when trying to continue execution of a core process.
-var ErrContinueCore = errors.New("can not continue execution of core process")
+	// ErrContinueCore is returned when trying to continue execution of a core process.
+	ErrContinueCore = errors.New("can not continue execution of core process")
+
+	// ErrChangeRegisterCore is returned when trying to change register values for core files.
+	ErrChangeRegisterCore = errors.New("not supported")
```

Comment:
While we're at it let's add a more descriptive error message, specifically mentioning this action is not supported in a core process.